src/lnclient/FiatAmount.ts

```diff
@@ -0,0 +1,19 @@
+import { fiat } from "@getalby/lightning-tools";
+
+// TODO: move to Lightning Tools
+export class FiatAmount {
+  satoshi: Promise<number>;
+  constructor(amount: number, currency: string) {
+    this.satoshi = fiat.getSatoshiValue({
+      amount,
+      currency,
+    });
+  }
+}
+
+// Most popular fiat currencies
+export const USD = (amount: number) => new FiatAmount(amount, "USD");
```

Comment:
These are currently exported directly which muddies the root level exports. In js-lightning-tools they can be exported under `fiat`